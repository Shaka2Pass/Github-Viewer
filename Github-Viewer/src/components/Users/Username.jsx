import React from 'react';
import PropTypes from 'prop-types';

const Username = ({ username }) => (
  <header>
    <div>
      <p>{ username }Username</p>
    </div>
  </header>
);

Username.propTypes = {
  username: PropTypes.string.isRequired,
};

export default Username;

