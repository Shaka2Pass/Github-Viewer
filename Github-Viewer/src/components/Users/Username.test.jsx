import React from 'react';
import { shallow } from 'enzyme';
import Username from './Username';

describe('User', () => {
  it('matches a user', () => {
    const wrapper = shallow(<Username username="shaka2pass"/>);
    expect(wrapper).toMatchSnapshot();
  });
});

