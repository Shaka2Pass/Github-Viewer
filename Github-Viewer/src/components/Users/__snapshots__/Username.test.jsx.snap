// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User matches a user 1`] = `
<header>
  <div>
    <p>
      shaka2pass
    </p>
  </div>
</header>
`;
